src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -955,7 +950,6 @@ void WindowImplX11::setIcon(const Vector2u& size, const std::uint8_t* pixels)
     GC        iconGC = XCreateGC(m_display, m_iconPixmap, 0, &values);
     XPutImage(m_display, m_iconPixmap, iconGC, iconImage, 0, 0, 0, 0, size.x, size.y);
     XFreeGC(m_display, iconGC);
-    XDestroyImage(iconImage);
```

Comment:
This should not be removed.

Reply:
Thanks. I forgot them when reverting the rest of the XImage stuff.